zone/embparser_api.cpp

```diff
@@ -5762,6 +5763,26 @@ std::string Perl__convert_money_to_string(perl::hash table)
 	return Strings::Money(platinum, gold, silver, copper);
 }
 
+uint32 Perl__get_guild_favor(uint32 guild_id)
+{
+	return guild_mgr.GetGuildFavor(guild_id);
+}
+
+void Perl__set_guild_favor(uint32 guild_id, uint32 favor)
+{
+	guild_mgr.SetGuildFavor(guild_id, favor);
+}
+
+uint32 Perl__get_guild_member_favor(uint32 guild_id, uint32 character_id)
+{
+	return guild_mgr.GetGuildMemberFavor(guild_id, character_id);
+}
+
+void Perl__set_guild_member_favor(uint32 guild_id, uint32 character_id, uint32 favor)
+{
+	guild_mgr.SetGuildMemberFavor(guild_id, character_id, favor);
```

Comment:
While this directly updates the database, it will have to also update the in memory version and then update the guild db and in memory version of guild, otherwise the change will be lost/overwritten when/if the player performs an actually donation.  We will also have to send the appropriate packets so that the entire guild is updated with the changed favor